benefits/core/migrations/0002_data.py

```diff
@@ -156,6 +161,26 @@ def load_data(app, *args, **kwargs):
         enrollment_success_expiry_item_details=None,
     )
 
+    mst_veteran_verifier = EligibilityVerifier.objects.create(
+        name=os.environ.get("MST_VETERAN_VERIFIER_NAME", "VA.gov - Veteran (MST)"),
+        eligibility_type=mst_veteran_type,
```

Comment:
@thekaveman Should I just put some other variable here then? Like just stub it with `mst_senior_type`?

Reply:
No, I think it's fine that this PR closes both (and I see the reasoning) -- just want to make sure it fully closes both (LP config, KeyVaults, etc.)